techscreenr-angular/package-lock.json

```diff
@@ -209,6 +209,50 @@
         "tslib": "1.9.0"
```

Comment:
@amartinelli12 Should this be gitIgnored? Peter eventually wants to help and thought that package-lock was IDE specific?